### DISCUSSION TITLE
Beneficios de estudiante

### DISCUSSION THREAD

**Discussion by Juanjopinedab** [community/community#19245] [GitHub Education]
Buenas noches, acabo de solicitar los beneficios de pro de estudiante pero al cargar el comprobante solo deja tomar foto al documento desde la computadora y no queda bien, se pueden subir los documentos de otra manera?

**Comment by melvincwng** [comment]
@Juanjopinedab, you can upload an image of your student ID directly during the application for GitHub Education Student Developer pack.

Please refer to [this](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack) for more information.

**Comment by queenofcorgis** [comment]
Hi there - I've moved your Discussion over to our Education section as those folks will be better able to assist you 😄
